content/svenska/data_types/health_data/npc-statistics.md

```diff
@@ -10,19 +10,19 @@ menu:
 #### Dagligt antal tester vid NPC
 
 <div class="d-md-none alert alert-info">
-  Scroll the plot sideways to view all data
+  Skrolla grafen horisontellt för att se allt data.
 </div>
 <div class="plot_wrapper">
-     <div id="stacked-bar-chart"></div>
+  <div id="stacked-bar-chart"></div>
 </div>
 
 #### Kumulativt antal tester vid NPC
 
 <div class="d-md-none alert alert-info">
-  Scroll the plot sideways to view all data
+  Skrolla grafen horisontellt för att se allt data.
```

Comment:
Is it "all data" or "allt data"? I would have chosen the earlier without thinking, but maybe I have missed some deep discussions about how the wording should be?